Lib/test/test_sys.py

```diff
@@ -1087,6 +1088,11 @@ def test_implementation(self):
         self.assertEqual(sys.implementation.name,
                          sys.implementation.name.lower())
 
+        # https://peps.python.org/pep-0734
+        self.assertIsInstance(sys.implementation.supports_isolated_interpreters, bool)
+        if test.support.check_impl_detail(cpython=True):
+            self.assertIs(sys.implementation.supports_isolated_interpreters, True)
```

Comment:
You can use `assertTrue`:

```suggestion
            self.assertTrue(sys.implementation.supports_isolated_interpreters)
```